models/group.py

```diff
@@ -9,6 +9,8 @@
 from formula_one.validators.aspect_ratio import AspectRatioValidator
 from formula_one.validators.year_relation import YearRelationValidator
 
+from tinymce.models import HTMLField
```

Comment:
`HTMLField` being a third-party package needs to be grouped with 
```python
import swapper
from django.contrib.contenttypes import fields as contenttypes_fields
from django.db import models
```

Remember to leave a line after the group ends.

Reply:
Hey I made the change. Is it fine ?